docs/man_pages/project/creation/init-hybrid.md

```diff
@@ -1,40 +1,38 @@
 init hybrid
 ==========
 
-Usage | Syntax
+Usage | Synopsis
 ------|-------
 General | `$ appbuilder init hybrid [--appid <App ID>]`
-Initializes an existing Apache Cordova project for development in the current directory. 
 
-The operation creates one or more of the following AppBuilder-specific files, if missing:
+Initializes an existing Apache Cordova project for development in the current directory. <% if(isHtml) { %>If the directory contains an existing AppBuilder project (created with the Telerik AppBuilder extension for Visual Studio or synchronized from GitHub), the project retains any existing project configuration. In this case, you might want to manually set new unique values for the WP8ProductID and WP8PublisherID properties to avoid issues when running your app on device.  
+For more information about how to configure your project properties, see [appbuilder prop](../configuration/prop.html)<% } %> 
+
+### Options
+* `--appid` - Sets the application identifier for your app. 
+
+### Attributes
+* `<App ID>` must consist of at least three alphanumeric strings, separated by a dot (.). Each string must start with a letter. The application identifier corresponds to the Bundle ID for iOS apps and to the package identifier for Android apps. If not specified, the application identifier is set to `com.telerik.<current directory name>`.
```

Comment:
The app id described here is differently worded than the one in create command. Should we make them the same?